Shipr-Kubernetes/shipr-payment-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shipr-payment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: shipr-payment
+  template:
+    metadata:
+      labels:
+        app: shipr-payment
+    spec:
+      containers:
+      - name: shipr-payment
+        image: osamaazm/payment:v1
+        ports:
+        - containerPort: 8000
+        env:
+          - name: REDIS_HOST
+            value: "redis"
+          - name: REDIS_PORT
+            value: "6379"
```

Comment:
I think Payment service also requires other environment variables to work correctly, Any reason those are excluded?

Reply:
Added